source/VehicleCommander/mapconfig.xml

```diff
@@ -39,21 +39,21 @@
         <!--layer name="Bihsud Bridge and Environs" visible="true" type="LocalDynamicMapLayer" basemap="false">
             <datasetpath>../../BihsudBridgeVCP.mpk</datasetpath>
         </layer> -->
-        <layer name="COA Sketch" visible="true" type="Mil2525CMessageLayer" basemap="false">
+        <!--layer name="COA Sketch" visible="true" type="Mil2525CMessageLayer" basemap="false">
```

Comment:
We're focusing on Monterey, not San Francisco, so I'll make a new .xml file. It shouldn't be too hard (famous last words) using the Military Planner app.